milestone1/mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
Do we really need this file? We can start mvn by executing the command directly, right?

Reply:
We can delete it no problem. Spring created it when initializing the project